modules/post/multi/general/close.rb

```diff
@@ -10,11 +10,16 @@ def initialize(info = {})
       update_info(
         info,
         'Name' => 'Multi Generic Operating System Session Close',
-        'Description' => %q{ This module closes the specified session. This can be useful as a finisher for automation tasks },
+        'Description' => %q{This module closes the specified session. This can be useful as a finisher for automation tasks.},
         'License' => MSF_LICENSE,
         'Author' => [ 'hdm' ],
         'Platform' => %w[linux osx unix win],
-        'SessionTypes' => [ 'shell', 'meterpreter' ]
+        'SessionTypes' => [ 'shell', 'meterpreter' ],
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
```

Comment:
I don't know if we can guarantee this here, since it's just a generic session kill - depending on the context it's been called in, I guess it could have different side effects 🤔

Reply:
I'll change it to blank. `Stability` defaults to unknown when not specified, which is a reasonable default.